### DISCUSSION TITLE
在vue3+ts项目中根本无法使用啊，好垃圾

### DISCUSSION THREAD

**Discussion by yhuayang233** [bytedance/xgplayer#999] [Q&A]
![image](https://github.com/bytedance/xgplayer/assets/133995431/f3109e10-b24d-40f5-9f10-fd32b2850c21)
报错提示：
![image](https://github.com/bytedance/xgplayer/assets/133995431/fbde8a0c-1130-48d7-a225-c649c132a21d)
接下来把css文件也加上还是报错：
![image](https://github.com/bytedance/xgplayer/assets/133995431/27880149-a765-4ea7-9311-8a33a7499138)

**Comment by amihhs** [comment]
首先菜就好好学，不要什么都怪到垃圾上面。
有没有可能执行 `new Player` 的时候元素没有挂载完成

**Reply by sdzhongqian** [reply]
确实有文档，vue3+ts 播放加密hls-m3u8视频无法播放，我通过直接使用hls.js可以播放，但是还是有些小问题
